build.gradle

```diff
@@ -262,7 +262,8 @@ def setCommonTestConfig(Test task) {
                 "sun.security.jgss.*",
                 "sun.security.pkcs11.*",
                 "sun.security.smartcardio.*",
-                "sun.util.resources.provider.*"
+                "sun.util.resources.provider.*",
+                "org/opensearch/security/securityconf/impl/v6.*"
```

Comment:
Remove this exclusion, there is lots of code that could be the source of serious bugs if not tested properly.